<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaybillBrowser</class>
 <widget class="QWidget" name="WaybillBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1177</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt SQL Browser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableView" name="table">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="OrderLabel">
           <property name="text">
            <string>Sort by</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="OrderBox">
           <item>
            <property name="text">
             <string>Id</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Serial Number</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Title</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Produced</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>State</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Firstname</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Lastname</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>City</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Telefon</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="PreviewLabel">
           <property name="text">
            <string>Preview</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="SerNumCB">
           <property name="text">
            <string>Serial Number</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="TitleCB">
           <property name="text">
            <string>Title</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="FirstnameCB">
           <property name="text">
            <string>Firstname</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="LastnameCB">
           <property name="text">
            <string>Lastname</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="CityCB">
           <property name="text">
            <string>City</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="TelefonCB">
           <property name="text">
            <string>Telefon</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="StateCB">
           <property name="text">
            <string>State</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="ProducedCB">
           <property name="text">
            <string>Produced</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="FilterLabel1">
           <property name="text">
            <string>Filter 1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="FilterLine1"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="ConditionLabel">
           <property name="text">
            <string>Condition</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="ConditionBox">
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>==</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>!=</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>AND</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OR</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="FilterLabel2">
           <property name="text">
            <string>Filter 2</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="FilterLine2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="newRowBtn">
           <property name="text">
            <string>NewRow</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="modifyBtn">
           <property name="text">
            <string>Modify</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>Report</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="delRowBtn">
           <property name="text">
            <string>DelRow</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="createBtn">
           <property name="text">
            <string>Create</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="updateBtn">
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="insertRowAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Добавить запись</string>
   </property>
   <property name="statusTip">
    <string>Inserts a new Row</string>
   </property>
  </action>
  <action name="deleteRowAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Удалить запись</string>
   </property>
   <property name="statusTip">
    <string>Deletes the current Row</string>
   </property>
  </action>
  <action name="submitAction">
   <property name="text">
    <string>&amp;Submit All</string>
   </property>
   <property name="toolTip">
    <string>Submit Changes</string>
   </property>
  </action>
  <action name="revertAction">
   <property name="text">
    <string>&amp;Revert All</string>
   </property>
   <property name="toolTip">
    <string>Revert</string>
   </property>
  </action>
  <action name="selectAction">
   <property name="text">
    <string>S&amp;elect</string>
   </property>
   <property name="toolTip">
    <string>Refresh Data from Database</string>
   </property>
  </action>
  <action name="on_newDeviceAction">
   <property name="text">
    <string>&amp;NewDevice</string>
   </property>
   <property name="toolTip">
    <string>New Device</string>
   </property>
  </action>
  <action name="editRowAction">
   <property name="text">
    <string>Изменить запись</string>
   </property>
   <property name="toolTip">
    <string>Edit Row</string>
   </property>
  </action>
  <action name="copyRowAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Копировать вставкой</string>
   </property>
   <property name="toolTip">
    <string>Copy Row</string>
   </property>
   <property name="statusTip">
    <string>Deletes the current Row</string>
   </property>
  </action>
  <action name="updateTableAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Обновить таблицу</string>
   </property>
   <property name="toolTip">
    <string>Update Table</string>
   </property>
   <property name="statusTip">
    <string>Inserts a new Row</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>table</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
